simulation/simulation_linucb.py

```diff
@@ -1,54 +1,50 @@
-from striatum.storage import history
-from striatum.storage import model
-from striatum.bandit import linucb
-from striatum import simulation
-from striatum import rewardplot as rplt
 import numpy as np
+import matplotlib.pyplot as plt
+
+from striatum import simulation
+from striatum.storage import MemoryHistoryStorage, MemoryModelStorage
+from striatum.bandit import linucb
 from striatum.bandit.bandit import Action
```

Comment:
Do we need to change Action from striatum.bandit.bandit.Action to striatum.bandit.Action,
or open a new submodule striatum.base.Action and also define all the interfaces there?